src/opencensus/proto/metrics/v1/metrics.proto

```diff
@@ -91,11 +92,12 @@ message MetricDescriptor {
     CUMULATIVE_INT64 = 4;
 
     // Floating point cumulative measurement. The value can only go up, if
```

Comment:
The value is allowed to stay the same, right?

How about "the value cannot decrease"